File: C# .Net/JDI UI Framework/Tests/TutorialTests/Career/PageObjects/Pages/JobDescriptionPage.cs

using Epam.JDI.Core.Interfaces.Base;
using Epam.JDI.Web.Attributes;

namespace Epam.Tests.TutorialTests.Career.PageObjects
{
    public class JobDescriptionPage : JDI.Web.Selenium.Elements.Composite.WebPage
    {
        [FindBy(Css = ".form-constructor")]
        public AddCVForm addCVForm;

        [FindBy(Id = "captcha-input")]
        public IElement captcha;

    }
}

File: Java/Tests/jdi-uitest-tutorialtests/src/main/java/com/epam/jdi/site/epam/pages/JobDescriptionPage.java
package com.epam.jdi.site.epam.pages;

import com.epam.jdi.site.epam.sections.AddCVForm;
import com.epam.jdi.uitests.core.interfaces.base.IElement;
import com.epam.jdi.uitests.web.selenium.elements.composite.WebPage;
import com.epam.jdi.uitests.web.selenium.elements.pageobjects.annotations.simple.Css;

/**
 * Created by Roman_Iovlev on 10/22/2015.
 */
public class JobDescriptionPage extends WebPage {

    @Css(".section__sidebar") public AddCVForm addCVForm;

    @Css(".captcha-ui") public IElement captcha;

}
